acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/OptimisticSyncPostMergeAcceptanceTest.java

```diff
@@ -0,0 +1,125 @@
+import tech.pegasys.teku.test.acceptance.dsl.TekuNode;
+import tech.pegasys.teku.test.acceptance.dsl.tools.deposits.ValidatorKeystores;
+
+public class OptimisticSyncPostMergeAcceptanceTest extends AcceptanceTestBase {
+  private static final String NETWORK_NAME = "less-swift";
+
+  private final SystemTimeProvider timeProvider = new SystemTimeProvider();
+  private BesuNode eth1Node1;
+  private BesuNode eth1Node2;
+  private BesuNode eth1Node3;
+  private TekuNode tekuNode1;
+  private TekuNode tekuNode2;
+  private String eth1Node1Enode;
+
+  @BeforeEach
+  void setup() throws Exception {
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    eth1Node1 =
+        createBesuNode(
+            config ->
+                config
+                    .withMiningEnabled()
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json"));
+    eth1Node1.start();
+    eth1Node1Enode = eth1Node1.fetchEnodeUrl();
+    eth1Node2 =
+        createBesuNode(
+            config ->
+                config
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json")
+                    .withStaticNodes(eth1Node1Enode));
+    eth1Node2.start();
+    eth1Node3 =
+        createBesuNode(
+            config -> config.withMergeSupport(true).withGenesisFile("besu/preMergeGenesis.json"));
+    // eth1Node3 will never sync, we will start it later
+
+    final int totalValidators = 4;
+    final ValidatorKeystores validatorKeystores =
+        createTekuDepositSender(NETWORK_NAME).sendValidatorDeposits(eth1Node1, totalValidators);
+    tekuNode1 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node1, genesisTime)
+                    .withValidatorProposerDefaultFeeRecipient(
+                        "0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73")
+                    .withValidatorKeystores(validatorKeystores));
+    tekuNode1.start();
+    tekuNode2 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node2, genesisTime)
+                    .withInteropValidators(0, 0)
+                    .withPeers(tekuNode1));
+    tekuNode2.start();
+  }
+
+  @Test
+  void shouldSwitchToOptimisticSyncAfterMergeWhenExecutionEngineIsSyncing() throws Exception {
+    tekuNode2.waitForGenesis();
+    tekuNode2.waitForLogMessageContaining("MERGE is completed");
+    tekuNode1.waitForNonDefaultExecutionPayload();
+    tekuNode2.waitForNonDefaultExecutionPayload();
+
+    // Stopping eth1Node2
+    final String eth1Node2Alias = eth1Node2.getNodeAlias();
+    eth1Node2.stop();
+    // Set same network name to the eth1Node3 that eth1Node2 had
+    eth1Node3.setNodeAlias(eth1Node2Alias);
+    eth1Node3.start();
+    eth1Node3.waitForLogMessageContaining("Ethereum main loop is up");
```

Comment:
We seem to be trying to just reset eth1Node2's database to force it to resync?  We really want to hide the details of this from the AT class so we can just have a method like `eth1Node2.restartWithEmptyDatabase()`. I'd be very tempted to use `container.execInContainer` to just delete the database then restart the existing container rather than recreating one with the same alias.